src/aatoolbox/datasources/iri/iri_forecast_seasonal_precipitation_tercile.py

```diff
@@ -0,0 +1,184 @@
+"""Class to download and load IRI's seasonal forecast.
+
+Data is downloaded from `IRI's maproom
+<https://iridl.ldeo.columbia.edu/maproom/Global/Forecasts/NMME_Seasonal_Forecasts/Precipitation_ELR.html>`_
+
+For now only the tercile precipitation forecast has been
+implemented.
+"""
+
+import logging
+
+import requests
+import xarray as xr
+
+import aatoolbox.utils.raster  # noqa: F401
+from aatoolbox.datasources.datasource import DataSource
+from aatoolbox.utils.geoboundingbox import GeoBoundingBox
+
+logger = logging.getLogger(__name__)
+
+_MODULE_BASENAME = "iri"
+
+
+class IriForecast(DataSource):
+    """
+    Work with IRI's forecast data.
+
+    Parameters
+    ----------
+    iso3 : str
+        country iso3
+    geo_bounding_box: GeoBoundingBox
+        the bounding coordinates of the area that should
+        be included in the data.
+    """
+
+    def __init__(self, iso3, geo_bounding_box: GeoBoundingBox):
+        super().__init__(
+            iso3=iso3, module_base_dir=_MODULE_BASENAME, is_public=False
+        )
+        self._iso3 = iso3
```

Comment:
I don't think this is needed

Reply:
it is used for the filename. Unless we want to move away from having filenames start with their iso3?

Reply:
Sorry I should have explained: it's already made an instance variable in `Datasource` since it is so widely used